data/korath/korath missions.txt

```diff
@@ -257,3 +257,81 @@ mission "Small Scale Delivery"
 	on complete
 		payment 1000
 		dialog `You find a Korath who seems to have been waiting for you in the spaceport. They greet you by showing their palms, then outstretch their hands as if waiting to receive the package. You hand off the small delivery and they hand you a small payment of <payment>.`
+
+mission "Efret: Quarg Rescue"
+	boarding
+	repeat
+	invisible
+	deadline 1
+	source
+		government "Kor Efret"
+	on offer
+		"old quarg rep" = "reputation: Quarg"
+		# bring it low enough that capping this will result in war
+		"reputation: Quarg" <?= 1
+		conversation
+			branch "unfriendly quarg" # if you're already at war with them
+				"reputation: Quarg" < 0
+			`The Quarg have brought reinforcements, probably in reaction to a distress call by the <ship>. They hail you, with barely contained anger in their voice. "Remove yourself from hence, and we may yet forgive your aggression. The Kor Efret are under our protection! Think not even about capturing yonder vessel."`
+			choice
+				`  "I had not realized they were under your protection. I will leave at once."`
+				`  (Ignore the warning.)`
+					goto ignore
+				`  "I think you should leave. This is my prize and I'll take what I please."`
+					goto refuse
+			`  "Leave then anon, ere we change our mind."`
+				accept
+			label ignore
+			action
+				"reputation: Quarg" <?= -1
+			`  "So be it. You shall pay for your transgression against the Efret!" The Quarg begin powering up their weapons...`
```

Comment:
I didn't say or do anything. For all they know, I'm still making up my mind. No mention of them waiting any amount of time or repeating their message.

Reply:
You dont get to make up your mind, they have had enough patience with you already. But that should be mentioned

Reply:
Yes, if they're not going to wait for me, it should say that it does not take long for them to realise my decision.